packages/fxa-settings/src/constants/index.tsx

```diff
@@ -22,19 +22,25 @@ export enum ENTRYPOINTS {
   FIREFOX_FX_VIEW_ENTRYPOINT = 'fx-view',
 }
 
-export enum LINK {
-  AMO = 'https://addons.mozilla.org/',
-  FX_DESKTOP = 'https://www.mozilla.org/firefox/new/',
-  FX_MOBILE = 'https://www.mozilla.org/firefox/mobile/',
-  FX_SYNC = 'https://www.mozilla.org/firefox/sync/',
-  HUBS = 'https://hubs.mozilla.com/',
-  MDN = 'https://developer.mozilla.org/',
-  MONITOR = 'https://monitor.mozilla.org/',
-  MONITOR_PLUS = 'https://monitor.mozilla.org/#pricing',
-  POCKET = 'https://getpocket.com/',
-  RELAY = 'https://relay.firefox.com/',
-  VPN = 'https://vpn.mozilla.org/',
-}
+const getMonitorLinkByHost = () => {
+  return window.location.host === 'accounts.stage.mozaws.net'
+    ? 'https://stage.firefoxmonitor.nonprod.cloudops.mozgcp.net/'
+    : 'https://monitor.mozilla.org/';
+};
+
+export const LINK = {
+  AMO: 'https://addons.mozilla.org/',
+  FX_DESKTOP: 'https://www.mozilla.org/firefox/new/',
+  FX_MOBILE: 'https://www.mozilla.org/firefox/mobile/',
+  FX_SYNC: 'https://www.mozilla.org/firefox/sync/',
+  HUBS: 'https://hubs.mozilla.com/',
+  MDN: 'https://developer.mozilla.org/',
+  MONITOR: getMonitorLinkByHost(),
+  MONITOR_PLUS: 'https://monitor.mozilla.org/#pricing',
```

Comment:
I think Monitor Plus is also different in stage vs prod? I don't know how much that matters though. The other is more important for whenever they get `prompt=none` set up.

Reply:
Had just realized that I missed MONITOR_PLUS. Will update for stage check!